<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelStyleSql</class>
 <widget class="QWidget" name="LabelStyleSql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabelStyleSql</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="idEdit">
         <property name="placeholderText">
          <string>id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bsizeEdit">
         <property name="placeholderText">
          <string>border-size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bcolorEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>border-color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="bradiusEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>border-radius</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="backcolorEdit">
         <property name="placeholderText">
          <string>background-color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="colorEdit">
         <property name="placeholderText">
          <string>color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="insertBt">
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryBt">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBt">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="renewBt">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="testBt">
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="testLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
